portfolio/src/main/webapp/script.js

```diff
@@ -36,3 +36,29 @@ function addRandomFact() {
   funFactContainer.innerText = funFact;
   funFactContainer.style.backgroundColor = "whitesmoke";
 }
+
+function getComments() {
+  /* Clear comments before getting new comments so we don't just append comments everytime
+     the user changes how many comments they'd like to see */
+  clearComments();
+
+  const selectElement = document.getElementById("limit");
+  const commentLimit = selectElement.options[selectElement.selectedIndex].value;
+  fetch("/data?limit=" + commentLimit).then(response => response.json()).then((comments) => {
+    comments.forEach(addComment);
+  });
+}
+
+function addComment(comment) {
+  const commentContainer = document.getElementById('comment-container');
+  const node = document.createElement('li');
+  node.setAttribute("id", "comment");
```

Comment:
I'm not really an expert here but shouldn't we try to have unique ids for nodes? I *think* you could have this function signature into `addComment(comment, index)` and it'll give you the index in the array too which you could use to unique-ify the id.

Reply:
Done.